Translate the python agent README.md into English (#58)
naver/scavenger: scavenger-agent-python/README.md

from ...
```

### Limitations

> * The implementation of Graceful Shutdown is not automatically supported due to potential conflicts with existing frameworks. Instead, the `agent.shutdown()` function as shown below is supported, so you must register and use it yourself.

Reviewer: ```suggestion
* The implementation of Graceful Shutdown is not automatically supported due to potential conflicts with existing frameworks. Instead, the `agent.shutdown()` function as shown below is provided, so you must register it into your framework shutdown routine by yourself.
```

def hello_world():
    return 'Hello World!'
```

> * Pyc files are currently not supported because Python code must be read by the agent.

Reviewer: ```suggestion
* The direct Instrumentation for pyc files are currently not supported because Python code must be read by the scavenger agent.
```

Reviewer 2: In that case, how about updating Korean too?

Reviewer: good!!
